src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+
+  if (days < 3) {
+    return pricePerDay * days;
+  }
+
+  if (days >= 7) {
+    return pricePerDay * days - 50;
+  }
+
+  if (days >= 3) {
+    return pricePerDay * days - 20;
+  }
```

Comment:
You need only 2 if statements here. The function will stop after the return statement anyway.
```
if (...) {
  return ...
}

if (...) {
  return ...
}

return ...